### ISSUE TITLE
Popover should close if focus leaves the component

### ISSUE THREAD

**Issue by abeidahmed** [Ambiki/impulse_view_components#148]
This is primarily useful for keyboard users. If the focus leaves the component, the panel should be hidden.
